karma/test-directives.coffee

```diff
@@ -110,5 +115,60 @@ describe('directives:', () ->
       expect(item2.querySelector('.pod-item-value').textContent)
         .toContain('Corge')
     )
+
+    it('should draw the pod actions', ->
+      $rootScope.podData.actions = [{
+        case_id: 21,
+        type: 'foo',
+        name: 'Foo',
+        payload: {bar: 'baz'}
+      }, {
+        case_id: 21,
+        type: 'quux',
+        name: 'Quux',
+        payload: {corge: 'grault'}
+      }]
+
+      el = $compile('<div cp-pod></div>')($rootScope)[0]
+      $rootScope.$digest()
+
+      action1 = el.querySelectorAll('.pod-action')[0]
+      action2 = el.querySelectorAll('.pod-action')[1]
+
+      expect(action1.textContent).toContain('Foo')
+      expect(action2.textContent).toContain('Quux')
+    )
+
+    it('should call trigger() when an action button is clicked', ->
+      $rootScope.podData.actions = [{
+        type: 'foo',
+        name: 'Foo',
+        case_id: 21,
+        payload: {a: 'b'}
```

Comment:
Ah, I see that in PR #45 you send the case id to the backend. Would you like the backend to send it back in every item and action?